examples/pxScene2d/src/Makefile

```diff
@@ -15,12 +15,15 @@ libs-glut: libpxscene-glut
 libs-dfbwindow: libpxscene-dfbwindow
 endif
 
+
 HAVE_WESTEROS := $(shell test -f ../external/westeros/Makefile.ubuntu && echo 1)
 
 ifneq ($(HAVE_WESTEROS),1)
 HAVE_WESTEROS := $(shell pkg-config --exists westeros-compositor && echo 1)
 endif
 
+HAVE_RT_REMOTE := $(shell test -f ../../../remote/librtRemote.so -o -f ../../../remote/librtRemote_s.a && echo 1)
```

Comment:
We should probably use a different means of determining whether or not to build with rtRemote support.  These libraries will be built for most platforms, but are not included in all.  Perhaps, we can use an environment variable here called SERVICE_MANAGER.  If we want to build with the service manager then we run: make SERVICE_MANAGER=1.  We can then test to see if this environment variable is set inside the Makefile.  If set then we will compile in pxServiceManager.cpp plus link in the rtRemote libs.